pxr/usd/usdPhysics/massProperties.h

```diff
@@ -0,0 +1,251 @@
+//
+// Copyright 2021 Pixar
+//
+// Licensed under the Apache License, Version 2.0 (the "Apache License")
+// with the following modification; you may not use this file except in
+// compliance with the Apache License and the following modification to it:
+// Section 6. Trademarks. is deleted and replaced with:
+//
+// 6. Trademarks. This License does not grant permission to use the trade
+//    names, trademarks, service marks, or product names of the Licensor
+//    and its affiliates, except as required to comply with Section 4(c) of
+//    the License and to reproduce the content of the NOTICE file.
+//
+// You may obtain a copy of the Apache License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the Apache License with the above modification is
+// distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. See the Apache License for the specific
+// language governing permissions and limitations under the Apache License.
+//
+#ifndef PXR_USD_USD_MASS_PROPERTIES_H
+#define PXR_USD_USD_MASS_PROPERTIES_H
+
+#include "pxr/pxr.h"
+#include "pxr/usd/usdPhysics/api.h"
+#include "pxr/usd/usd/common.h"
+
+#include "pxr/base/gf/vec3f.h"
+#include "pxr/base/gf/quatf.h"
+#include "pxr/base/gf/rotation.h"
+#include "pxr/base/gf/matrix3f.h"
+#include "pxr/base/gf/matrix4f.h"
+
+PXR_NAMESPACE_OPEN_SCOPE
+
+GfQuatf IndexedRotation(uint32_t axis, float s, float c)
```

Comment:
@AlesBorovicka , since this is a public header, our coding conventions require that every public function, type, and class, be prefixed with the ProperCase module name, so `UsdPhysicsIndexedRotation()` etc.  We do not allow module namespaces, _but_ do allow a nesting **class** of static methods, if you prefer.  So `UsdPhysicsMass::IndexedRotation()` etc, if that seems more appealing.  Thanks!